Testscripts/Linux/STORAGE-PartitionDisks.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+
+ICA_TESTRUNNING="TestRunning"
```

Comment:
these should no longer be use in lisav2?

Reply:
+send utils.sh to VM, source it and use UtilsInit